airgun/entities/contenthost.py

```diff
@@ -209,3 +208,23 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+
+
+@navigator.register(ContentHostEntity, 'Errata Details')
+class NavigateToErrataDetails(NavigateStep):
+    """Navigate to Errata details screen.
+
+    Args:
+        entity_name: name of content host
+        errata_id: id of errata
+    """
+    VIEW = ErrataDetailsView
+
+    def prerequisite(self, *args, **kwargs):
+        return self.navigate_to(
+            self.obj, 'Edit', entity_name=kwargs.get('entity_name'))
+
+    def step(self, *args, **kwargs):
+        errata_id = kwargs.get('errata_id')
+        self.parent.errata.search(errata_id)
```

Comment:
In previous version, that line was:
```python
view.errata.search(errata_id, lce=environment)
```
In this implementation we don't have to pass `lce`?

Reply:
Done! I missed that